save-demo/db/tables/dependency.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd">
+
+    <changeSet id="dependency-1" author="sanyavertolet" context="dev or prod">
+        <createTable tableName="dependency">
+            <column name="id" type="bigint" autoIncrement="true">
+                <constraints primaryKey="true" nullable="false"/>
+            </column>
+            <column name="master_id" type="bigint">
+                <constraints nullable="false"/>
+            </column>
+            <column name="slave_id" type="bigint">
+                <constraints nullable="false"/>
+            </column>
+        </createTable>
+
+        <addForeignKeyConstraint baseColumnNames="master_id"
+                                 baseTableName="dependency"
+                                 constraintName="fk_dependency_master"
+                                 referencedColumnNames="id"
+                                 referencedTableName="tool"
+                                 onDelete="CASCADE"/>
+
+        <addForeignKeyConstraint baseColumnNames="slave_id"
```

Comment:
can we change column name to `worker_id`, just for case.

Reply:
Done